src/library/export/coverartcopywizard.h

```diff
@@ -0,0 +1,32 @@
+// CoverArtCopyWizard handles copying the cover art of the track to a where track located.
+
+#pragma once
+
+#include <QScopedPointer>
+#include <QString>
+
+#include "library/export/coverartcopyworker.h"
+#include "library/export/dlgcoverartcopy.h"
+#include "preferences/usersettings.h"
+
+class CoverArtCopyWizard : public QObject {
+    Q_OBJECT
+  public:
+    CoverArtCopyWizard(QWidget* parent,
+            const QImage& coverArtImage,
+            const QString& coverArtCopyPath)
+            : m_parent(parent),
+              m_coverArtImage(coverArtImage),
+              m_coverArtCopyPath(coverArtCopyPath) {
+    }
+    virtual ~CoverArtCopyWizard() = default;
+
+    bool copyCoverArt();
+
+  private:
+    QWidget* m_parent;
+    QImage m_coverArtImage;
+    QString m_coverArtCopyPath;
+    QScopedPointer<DlgCoverArtCopy> m_dialog;
```

Comment:
this can become a parented_ptr
else the ownership is at the QObject tree and the QScopedPointer

Reply:
I think we should remove the dialog part from this PR

Reply:
Fair Enough, when we decided to make it smaller, I wasn't sure if we should remove the dialog part or not. Removing it would be smaller and ofc better 👍